.gitignore

```diff
@@ -60,3 +60,6 @@ yarn-error.*
 
 # local env files
 .env*.local
+
+# Auto-generated icons
+packages/design-system-react/src/components/icon/icons/
```

Comment:
These icon files are automatically generated by SVGR during the build process and don't need to be tracked in git.